packages/react/src/primitives/shared/__tests__/styleUtils.test.tsx

```diff
@@ -269,8 +267,8 @@ describe('convertStylePropsToStyleObj: ', () => {
 
 describe('useNonStyleProps: ', () => {
   it('should return an object containing only the non style props', () => {
-    const { result } = renderHook(() => useNonStyleProps(props));
-    const nonStyleProps = result.current;
+    const { result } = renderHook(() => useStyles(props, {}));
```

Comment:
nit: I think you can destructure this instead?
```suggestion
    const { result: {current: {nonStyleProps}} } = renderHook(() => useStyles(props, {}));
```